root/static/scripts/tests/edit/utility/getUpdatedTrackArtists.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2024 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import test from 'tape';
+
+import getUpdatedTrackArtists from
+  '../../../edit/utility/getUpdatedTrackArtists.js';
+import {genericArtist} from '../../utility/constants.js';
+
+test('getUpdatedTrackArtists', function (t) {
+  const name = (
+    name: string,
+    gid: string,
+    joinPhrase: string = '',
+    creditedAs: string = '',
+  ) => {
+    const artist = {...genericArtist, name, gid};
+    return {artist, joinPhrase, name: creditedAs};
+  };
+
+  const testCases = [
+    {
+      desc: 'artist change, track matches old',
+      track: [name('Old', '1')],
+      oldRel: [name('Old', '1')],
+      newRel: [name('New', '2')],
+      want: [name('New', '2')],
+    },
+    {
+      desc: 'artist change, track doesn\'t match old',
```

Comment:
You can use unicode apostrophes rather than escaping the ASCII ones here.

Reply:
The main benefit of that is if the test fails, you can grep for the string more easily, since what is printed will match the source code.

Reply:
Makes sense; thanks for the explanation. I've updated this.